storage/local/actor.ts

```diff
@@ -1,24 +1,137 @@
-import { RequestType, type State } from "./state"
+import address from "~packages/util/address"
+import type { HexString } from "~typing"
+
+import {
+  Eip712Status,
+  RequestType,
+  TransactionStatus,
+  TransactionType,
+  type Erc4337TransactionFailed,
+  type Erc4337TransactionLogMeta,
+  type Erc4337TransactionRejected,
+  type Erc4337TransactionReverted,
+  type Erc4337TransactionSent,
+  type Erc4337TransactionSucceeded,
+  type Request,
+  type RequestLogMeta,
+  type State
+} from "./state"
 
 /**
  * @dev StateActor mutates the state in-place, which can work with zustand and immer seamlessly.
  */
 export class StateActor {
   public constructor(private state: State) {}
```

Comment:
I want to discuss with @pilagod:
If we can use call-by-reference parameters here, like passing an `ObservableStorage`, instead of call-by-value.
This would enable methods such as `deleteTransactionLog()` and `deleteEip712Log()`, etc.
What do you think?

Reply:
Sure, we can do it if we need. One reminding is that we should focus more on the integrity of operation, instead of just wrapping expressions that is simple enough to do directly on language itself.

Reply:
Thank you! I got it!
Please have pilagod evaluate whether it’s suitable to proceed with the rewrite or not!